bigquery-storage/api/BigQueryStorage.Samples/BigQueryStorage.Samples.csproj

```diff
@@ -4,6 +4,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup> 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.BigQuery.Storage.V1" Version="2.9.0" />  
+    <PackageReference Include="Google.Cloud.BigQuery.Storage.V1" Version="2.9.0" />
+    <PackageReference Include="Google.Protobuf" Version="3.20.0" />  
```

Comment:
When we have released a new version of Google.Cloud.BigQuery.Storage.V1 that depens on Google.Protobuf >=3.20 we won't need this (the dependency is through GAX). Can you please add a reminder for yourself to clean this up when that happens? I'll try to remember as well.